src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, {
+  Dispatch,
+  SetStateAction,
+  useEffect,
+  useState,
+} from 'react';
+import cn from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo | null,
+  changeTodo: Dispatch<SetStateAction<Todo | null>>
```

Comment:
we can just pass prop like this
```suggestion
  closeModal: () => void
```